config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -59,7 +59,7 @@
 # Rails.application.config.active_record.automatic_scope_inversing = true
 
 # Raise when running tests if fixtures contained foreign key violations
-# Rails.application.config.active_record.verify_foreign_keys_for_fixtures = true
+Rails.application.config.active_record.verify_foreign_keys_for_fixtures = true
```

Comment:
This is for alerts on the missing test fixture. I assume this only applies to minitest's fixture.